wwwroot/test/hotspots_overview.geojson

```diff
@@ -0,0 +1,271 @@
+{
+  "type": "FeatureCollection",
+  "features": [
+    {
+      "type": "Feature",
+      "properties": {
+        "marker-color": "#999",
+        "marker-size": "large",
+        "marker-symbol": "circle",
+        "rc-type": "hotspot",
+        "rc-sector": "agriculture",
+        "story-id": "g-7bc92b7c2b8ce676bfab4d3b1abc5d0e"
```

Comment:
This is the 'old' gist, should this be updated with the one Maarten made, or perhaps copy his changes into this gits